docs/topics/certificaterotation.md

```diff
@@ -0,0 +1,57 @@
+# Cluster certificate rotation
+
+Instructions on rotating TLS CA and certificates for an AKS Engine cluster.
+
+## Prerequesites
+
+- The etcd members MUST be in a healthy state before rotating the CA and certs (ie. `etcdctl cluster-health` shows all peers are healthy and cluster is healthy).
+- The apimodel file reflecting the current cluster configuration and a working ssh private key that has root access to all nodes. The apimodel file is persisted at AKS Engine template generation time, by default to the _output/ child directory from the working parent directory at the time of the aks-engine invocation.
+
+<a name="preparation"></a>
+
+## Preparation
+
+**CAUTION**: Rotating certificates can break component connectivity and leave the cluster in an unrecoverable state. Before performing any of these instructions on a live cluster, it is preferrable to backup your cluster state and migrate critical workloads to another cluster.
+
+## Rotation
+
+**CAUTION**: Rotating certificates will cause cluster downtime.
+
+run `aks-engine rotate-certs`. For example:
+
+```bash
+CLUSTER="<CLUSTER_DNS_PREFIX>" && bin/aks-engine rotate-certs --apimodel _output/${CLUSTER}/apimodel.json --client-id "<YOUR_CLIENT_ID>" --client-secret "<YOUR_CLIENT_SECRET>" --location <CLUSTER_LOCATION> --master-FQDN ${CLUSTER}.<CLUSTER_LOCATION>.cloudapp.azure.com --ssh _output/${CLUSTER}-ssh --subscription-id "<YOUR_SUBSCRIPTION_ID>" -g ${CLUSTER} -o _output/${CLUSTER}
```

Comment:
This command needs to change to `api-model` with a hyphen now. My fault.

Reply:
fixed

Reply:
Also, could you break this up with backslashes into several shorter lines? Currently it renders into markdown as one long line with a horizontal scrollbar.